### ISSUE TITLE
Aerial  constantly asking for permission to access macOS screensaver.  Did macOS 14.1 break Aerial again?

### ISSUE THREAD

**Issue by RGM1956** [JohnCoates/Aerial#1342]
After updating macOS to 14.1, Aerial or Aerial Companion constantly asks for permission to access macOS screensavers.  However, an affirmative response does not stick after reboot.

Did macOS 14.1 break Aerial again?

2021 24" 4-Port M1-iMac
macOS 14.1
Single, built-n monitor
Aerial Companion 1.5.2
Aerial 3.3.4

**Comment by glouel** [comment]
Hey @RGM1956 

Yeah it's one of those things, you need to go to System Settings and explicitly give "Full Disk Access" to Companion to make that disappear. 

Allowing when prompted is temporary sadly, Apple doesn't make those choices stick for some reason, which is a real pain.

**Comment by RGM1956** [comment]
Think, maybe, it’s intentional?

**Comment by RGM1956** [comment]
And now another new item.  Yes,  gave Aerial Companion Full Disk Access, but despite also setting it to restart (motion wallpaper) at startup, it (now) isn’t.

**Comment by glouel** [comment]
Which part doesn't restart ?

Does Aerial Companion show up in your menu bar at reboot, but it doesn't start playback, or does Aerial Companion NOT show up in your menu bar at reboot, and when you launch it manually, playback starts again ?

**Comment by RGM1956** [comment]
No, Companion no longer appears in the menu bar even though the app is set to start at login. I had to manually add Companion to Login Items” in order to get it to work. On Nov 4, 2023, at 10:19 AM, Guillaume Louel ***@***.***> wrote:﻿
Which part doesn't restart ?
Does Aerial Companion show up in your menu bar at reboot, but it doesn't start playback, or does Aerial Companion NOT show up in your menu bar at reboot, and when you launch it manually, playback starts again ?

**Comment by glouel** [comment]
![Capture d’écran 2023-11-07 à 11 28 51](https://github.com/JohnCoates/Aerial/assets/37544189/79be1a93-5f4c-4d8d-b62e-009f5a369fc3)

@RGM1956 are both selected as such here ? Not just the bottom one ? 

If they are, I'll try to have a look when I can but that might be a few days.

**Comment by RGM1956** [comment]
They are

**Comment by glouel** [comment]
Try changing "At Startup" to "Manually", then back to "At Startup". It's likely they broke something again with launch agents.

**Comment by RGM1956** [comment]
That fixed it.  Thanks,
